twine/commands/upload.py

```diff
@@ -57,7 +72,6 @@ def skip_upload(
 def _make_package(
     filename: str, signatures: Dict[str, str], upload_settings: settings.Settings
 ) -> package_file.PackageFile:
-    """Create and sign a package, based off of filename, signatures and settings."""
```

Comment:
@meowmeowmeowcat Why was this and the docstring for `check._check_file` deleted? If anything, I'm inclined to require docstrings on private functions and methods, since they often have non-trivial logic.

Reply:
I think it's because I just want to pass the `flake8` test, so I only add docstrings to public functions first. If you want, I can take some time and add docstrings to private functions. (Or do you prefer adding them by yourself?) :)

Reply:
Good instinct to limit the scope, and at this point I'd rather add private docstrings in a separate PR. However,  it seems like you deleted these docstrings. Were they causing flake8 to fail?

Reply:
> However, it seems like you deleted these docstrings. Were they causing flake8 to fail?

I think they were not causing flake8 to fail. Only missing public docstrings will cause flake8 to fail.

Reply:
Okay. With that in mind, I've restored them.